sceptre/resolvers/stack_output.py

```diff
@@ -69,16 +78,19 @@ def _get_stack_outputs(self, stack_name):
             else:
                 raise e
         else:
-            outputs = response["Stacks"][0]["Outputs"]
+            outputs = response["Stacks"][0].get("Outputs", None)
```

Comment:
Is there a reason or project conventions to refer to that show why throwing exceptions is preferred instead of checking a key exists before proceeding?

Reply:
If you look through the code there is a tendency to lean toward try/except when working with dicts, although it is not exclusively this way. Whilst the initial code submitted isn't wrong there is a slight implicit assumption with .get() that the key will exist, then needs to check which adds an unnecessary condition. If there is no need to give a default value, and there is a pretty obvious exception being handled (`KeyError`) then try/except result in more readable code.

Reply:
Resolved. Moved the code to be in 1 try catch to be cleaner but follows same as above